Classes_Homework5/exec_utils/configs.ini

```diff
@@ -0,0 +1,40 @@
+[LABELS]
+news_label = News
+ad_label = Private ad
+recipe_label = Recipe
+
+[FORMATS]
+date_format = %d/%m/%Y
+date_time_format = %d/%m/%Y %H.%M
+
+[INPUTS]
+city_msg = Please type in city name and hit ENTER:
+news_msg = Please type in news text and hit ENTER:
+ad_msg = Please type in your ad text and hit ENTER:
+ad_date = Please type in expiration date for your ad in the following format: ${FORMATS:date_format}
+recipe_msg = Please add your recipe text
+recipe_calories = How many calories is that?
+proceed_msg = Your section written to ${PATHS:target_file}! Would you like to add another section? (Y/N)
+
+[MESSAGES]
+message_start = Valid until:
+message_end = day(s) left
+fitness1 = Fitness tip: 0 calories, really? Is it water? Take 2l of this daily)
+fitness2 = Fitness tip: nice, light healthy food!
+fitness3 = Fitness tip: oh, that's a lot of calories! Don't forget to work out!
+calories_measure = Kcal
+goodbye = Thank you and see you again!
+
+[RESTRICTIONS]
+max_size = 256
+n = -
```

Comment:
This argument is not clear at 1st sight.

Reply:
I know) That's one in one of further commits i've added comment to this config.